test/ModuleInterface/IndexWhileBuilding.swiftinterface

```diff
@@ -0,0 +1,39 @@
+// swift-interface-format-version: 1.0
+// swift-module-flags: -module-name IndexWhileBuilding
+
+// REQUIRES: OS=macosx
```

Comment:
Why does the test require macOS?

Reply:
Hmm, it shouldn't. I adapted this from `test/ModuleInterface/ExplicitInterfaceBuild.swiftinterface`, which also claims to require macOS, but I don't see anything in it that should either. Maybe that one was a mis-copy of `test/ModuleInterface/ExplicitInterfaceObjc.swiftinterface`?

I've removed it from my test; let's see what happens.